docs/containers/balance-calculator/orchestration/accrual-balance-calculation.md

```diff
@@ -0,0 +1,565 @@
+      - [Scenario 6: Move an existing `TimeEntry` spanning two contiguous days a single day](#scenario-6--move-an-existing--timeentry--spanning-two-contiguous-days-a-single-day)
+      - [Scenario 7: Delete an existing `TimeEntry` spanning two contiguous days](#scenario-7--delete-an-existing--timeentry--spanning-two-contiguous-days)
+      - [Scenario 8: Move an existing `TimeEntry` spanning a single day to a different single day](#scenario-8--move-an-existing--timeentry--spanning-a-single-day-to-a-different-single-day)
+
+## Updating Accrual balance
+One of the key properties on an accrual record is the balance. Each instance represents the running total of all contributions up to a given date. When `TimeEntry` events are received then they are used to identify which type of `accrual` records they relate to and also to identify which date(s) are to have their balances calculated. 
+
+**Note** 
+- that the mechanism for identifying an `accrual` record type based on the data in a TimeEntry is outside of the scope of this document (see [Out of Scope](#out-of-scope))
+- there will be one accrual record for each date in the current (effective) Annualised Hours Agreement. See [Out of Scope](#out-of-scope) for more information
+- In the user interface when viewing Accrual balances the end user will be presented with an eroded view of their balances. The translation from the running total captured in the storage model to that eroded view will be dealt with in a separate design
+
+A key part of an `accrual` record is its set of `contribution` records.  A contribution references a `time_entry` record and it also records how many hours of work that `time_entry` contributes to the `accrual` record's balance. 
+
+By combining all `contribution` records for a given `accrual` record (as there could be multiple `TimeEntry` events spread over a single day) Accruals can work out the balance for a given `accrual` record. This process is described in the following sections.
+
+## How to calculate `contribution.value`
+When a new `TimeEntry` is received then Accruals must recalculate effected balances. Depending on how many contiguous days the new `TimeEntry` covers and whether or not the `TimeEntry` represents an update (typically to move time around) or a deletion then the action required is different.
+
+### Single day
+If the date portions of the `TimeEntry.actualStartTime` and `TimeEntry.actualEndTime` are for the same date then this `TimeEntry` covers a single day.
+
+`value` = `TimeEntry.actualEndTime` - `TimeEntry.actualStartTime`
+
+### Overlapping day
+If the date portions of the `TimeEntry.actualStartTime` and `TimeEntry.actualEndTime` are for the different dates then this `TimeEntry` covers multiple days. The assumption is that the days are contiguous and that typically no more than two days are spanned. 
```

Comment:
It’s definitely implied by the new GMT to BST section (_sick_ diagrams btw), but is it worth having a super clear statement here saying words to the effect of ‘when apportioning accruals to days for overlapping shifts we use UK time (inclusive of daylight savings)’

Apologies if this already exists somewhere and I’ve missed it

Reply:
Addressed under the GMT to BST section - now linking out to a Confluence page written by @stephenkaye-ho around the use of ISO 8061